src/libs/antares/study/parts/thermal/cluster.cpp

```diff
@@ -137,7 +137,8 @@ Data::ThermalCluster::ThermalCluster(Area* parent) :
     forcedLaw(thermalLawUniform),
     plannedLaw(thermalLawUniform),
     PthetaInf(HOURS_PER_YEAR, 0),
-    costsTimeSeries(1, CostsTimeSeries())
+    costsTimeSeries(1, CostsTimeSeries()),
+    ramping()
```

Comment:
This constructor is called by default, there is no need to call it explicitly here

Reply:
removed in commit [77daa36](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1817/commits/77daa3606908d6d45a008d5e61288b106232a6e3)